upup/pkg/kutil/kubecfg_builder.go

```diff
@@ -194,7 +195,14 @@ func (c *KubeconfigBuilder) DeleteKubeConfig() {
 	c.execKubectl("config", "unset", fmt.Sprintf("users.%s", c.Context))
 	c.execKubectl("config", "unset", fmt.Sprintf("users.%s-basic-auth", c.Context))
 	c.execKubectl("config", "unset", fmt.Sprintf("contexts.%s", c.Context))
-	fmt.Printf("Deleted kubectl config for %s\n", c.Context)
+	config, err := clientcmd.LoadFromFile(c.KubeconfigPath)
+	if err != nil {
+		fmt.Printf("Error reading kube config.")
```

Comment:
Might be better if we add a bit more context, e.g. `Failed to unset kubectl current context: error reading kube config.`. Though this isn't really an error path that should come up ever anyway...